api/test/queries.test.js

```diff
@@ -503,15 +503,15 @@ describe('queries', () => {
       })
 
       describe('lt: less than', () => {
-        it('filters out results where the selected field has a value less than the selected value', async () => {
+        it('returns dwellings where the field is less than the given value', async () => {
```

Comment:
There are a few tests that use the `'filters out results'` description string, so best rename them all.